ocs_ci/ocs/resources/pod.py

```diff
@@ -143,6 +145,59 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
+    def run_io(
+        self, storage_type, size, io_direction='rw', rw_ratio=75,
+        jobs=1, runtime=60,
+    ):
+        """
+        Execute FIO on a pod
+        This operation will run in background and will store the results in
+        'self.thread.result()'.
+        In order to wait for the output and not continue with the test until
+        FIO is done, call self.thread.result() right after calling run_io.
+        See tests/manage/test_pvc_deletion_during_io.py::test_run_io
+        for usage of FIO
+
+        Args:
+            storage_type (str): 'fs' or 'block'
+            size (str): Size in MB, e.g. '200M'
```

Comment:
should one pass '200' or '200M'? Maybe it would be better to have this arg as an integer

Reply:
done

Reply:
Actually, you can also pass 200GB and not MB so I don't want to overcomplicate this.
In case we will see this needs change we can change